### PR TITLE
Add doc for CEP section in data calc

### PR DESCRIPTION
**Pull request by crbrox**
@fgalan 
@dmoranj

### CODE REVIEW

File: docs/topics/data_calculation.md [hunk 1]
```diff
@@ -108,7 +108,109 @@ the [specification](https://github.com/telefonicaid/iotagent-node-lib/blob/maste
 For those data values that doesn't come from devices, but from external systems, and for those cases when the sources of
 context information to calculate those values are multiple, the [CEP](../cep.md) can be used to make some of those calculations.
 
-// @CARLOS: I GUESS THIS IS CORRECT, BUT I'M NOT COMPLETELY SURE. IF IT IS, SOME EXAMPLES WOULD BE APPRECIATED
+The CEP gets a notification cointaining a configurable set of attributes (as set in the subscription) and that allows using more fields than the ones sent by device. The notification mechanism allows to send the previous value of an attribute as metadata `previousValue` (see [metadata in notifications](http://fiware-orion.readthedocs.io/en/master/user/metadata/index.html#metadata-in-notifications)). So, as an example, we can think of an entity that periodically sends its coordinates and the time of the measure. We could calculate its average velocity by means of a rule that updates its field `velocity`.
```
Comment (crbrox) [inline docs/topics/data_calculation.md]: OK

Comment (fgalan) [inline docs/topics/data_calculation.md]: Fixed in 9f47e9e491022c3ef04b80ea85608cec849fc62e

**Comment by fgalan** [comment]
LGTM. Passing the ball to @dmoranj to final LGTM + merge + delete branch

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**